### DISCUSSION TITLE
Why "Copied history calculation" and "Copied history result" arent available for translation?

### DISCUSSION THREAD

**Discussion by inson1** [clementwzk/OpenCalc#375] [Q&A]
in [app/src/main/java/com/darkempire78/opencalculator/HistoryAdapter.kt](https://github.com/Darkempire78/OpenCalc/blob/c73c6e2ec7aa7d8f2207867f22376c5b822531e4/app/src/main/java/com/darkempire78/opencalculator/HistoryAdapter.kt#L153)
+"Copied result"
and maybe more?

I know that they are just label names, but shouldnt be also translated? User can still see them - " For display to the user, it also has a label."

**Comment by clementwzk** [comment]
It's a mistake

**Comment by inson1** [comment]
@inson1 ok, I will fix it
